glaciers/glaciers.ipynb

```diff
@@ -265,50 +168,21 @@
    "source": [
     "## Dashboard\n",
     "\n",
-    "The code and plots above should be fine for exploring this data in a notebook, but let's go further and make a shareable dashboard using [Panel](http://panel.pyviz.org).  Panel lets us add arbitrary custom functionality, such as a button to reset the selections by calling `clear_selections` which sets the selection_expr to `None`:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "pn.extension()\n",
-    "\n",
-    "def clear_selections(event):\n",
-    "    ls.selection_expr = None\n",
-    "\n",
-    "clear_button = pn.widgets.Button(name='Clear selection', align='center')\n",
-    "clear_button.param.watch(clear_selections, 'clicks');"
+    "The code and plots above should be fine for exploring this data in a notebook, but let's go further and make a dashboard using [Panel](http://panel.pyviz.org)."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "And we can add static text, Markdown, or HTML items like a title, instructions, and logos:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "title       = '<div style=\"font-size:35px\">World glaciers explorer</div>'\n",
-    "instruction = 'Box-select on each plot to subselect; clear selection to reset.<br>'\n",
-    "oggm_url    = 'https://raw.githubusercontent.com/OGGM/oggm/master/docs/_static/logos/oggm_s_alpha.png'\n",
-    "oggm_logo   = '<a href=\"https://oggm.org\"><img src=\"{0}\" width=170></a>'.format(oggm_url)\n",
-    "pn_url      = 'http://panel.holoviz.org/_static/logo_stacked.png'\n",
-    "pn_logo     = '<a href=\"https://panel.holoviz.org\"><img src=\"{0}\" width=140></a>'.format(pn_url)"
+    "We can add static text, Markdown, or HTML items like a title, instructions, and logos."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "If you want detailed control over the formatting, you could define these items in a separate [Jinja2 template](http://bokeh.pydata.org/en/latest/docs/user_guide/server.html#building-bokeh-applications).  But here, let's put it all together using Panel Row and Column objects, which can display objects and plots from many different libraries, including the HoloViews objects used here.  You'll then get an app with widgets and plots usable from within the notebook:"
+    "If you want detailed control over the formatting, you could define these items in a separate [Jinja2 template](http://bokeh.pydata.org/en/latest/docs/user_guide/server.html#building-bokeh-applications).  But here, let's put it all together using the `panel.pane` module, which can display many python objects and plots from many different libraries.  You'll then get an app with widgets and plots usable from within the notebook:"
```

Comment:
Let's link to Panel's site since it has a how-to on custom templates.

```suggestion
    "If you want detailed control over the formatting, you could define these items in a separate [Jinja2 template](https://panel.holoviz.org/how_to/templates/template_custom.html).  But here, let's put it all together using the `panel.pane` module, which can display many python objects and plots from many different libraries.  You'll then get an app with widgets and plots usable from within the notebook:"
```